components/gitpod-db/src/tables.ts

```diff
@@ -201,7 +201,6 @@ export class GitpodTableDescriptionProvider implements TableDescriptionProvider
         {
             name: "d_b_cost_center",
             primaryKeys: ["id", "creationTime"],
-            deletionColumn: "deleted",
```

Comment:
I'd rather remove the whole entry for this table, as it is confusing to have it in the periodic deleter at all.

Reply:
Not for this, but for all entries here. Looks like most are outdated.

Reply:
maybe while you are at it also delete the other use less entries (e.g. `d_b_usage' below)